src/WebJobs.Extensions/Extensions/Timers/Scheduling/TimerSchedule.cs

```diff
@@ -19,6 +18,7 @@ public abstract class TimerSchedule
         /// <summary>
         /// Gets a value indicating whether intervals between invocations should account for DST.        
         /// </summary>
+        [Obsolete("This property is obsolete and will be removed in a future version. All TimerSchedule implementations should now handle their own DST transitions.")]
```

Comment:
Does this show up in the .NET Isolated mapping interfaces?

Reply:
No -- looks like we never added the `Schedule` to that payload. I think we were trying to map the fewest required properties that we could?

https://github.com/Azure/azure-functions-dotnet-worker/blob/main/extensions/Worker.Extensions.Timer/src/TimerInfo.cs